components/collective-page/hero/Hero.js

```diff
@@ -211,22 +211,41 @@ const Hero = ({ collective, host, isAdmin, onPrimaryColorChange, callsToAction,
                 )}
               </Flex>
               {host && collective.isApproved && !isEvent && (
-                <Container mx={1} color="#969ba3" my={2}>
-                  <FormattedMessage
-                    id="Collective.Hero.Host"
-                    defaultMessage="{FiscalHost}: {hostName}"
-                    values={{
-                      FiscalHost: <DefinedTerm term={Terms.FISCAL_HOST} />,
-                      hostName: (
-                        <LinkCollective collective={host}>
-                          <Span data-cy="fiscalHostName" color="black.600">
-                            {host.name}
-                          </Span>
-                        </LinkCollective>
-                      ),
-                    }}
-                  />
-                </Container>
+                <Fragment>
+                  <Container mx={1} color="#969ba3" my={2}>
+                    <FormattedMessage
+                      id="Collective.Hero.Host"
+                      defaultMessage="{FiscalHost}: {hostName}"
+                      values={{
+                        FiscalHost: <DefinedTerm term={Terms.FISCAL_HOST} />,
+                        hostName: (
+                          <LinkCollective collective={host}>
+                            <Span data-cy="fiscalHostName" color="black.600">
+                              {host.name}
+                            </Span>
+                          </LinkCollective>
+                        ),
+                      }}
+                    />
+                  </Container>
+                  {collective.connectedTo.length !== 0 && (
+                    <Container mx={1} color="#969ba3" my={2}>
```

Comment:
I just checked https://www.figma.com/file/GF1gQ7fWNfaOAFNwuLeAUl/%5BOC.com%5D-Hot-Fixes?node-id=1081%3A12923 and the margin left here should be `16px`. This should do it: 

```suggestion
                    <Container mx={1} color="#969ba3" my="12px">
```